kubernetes-client/src/test/java/io/fabric8/kubernetes/client/utils/internal/OptionalDependencyWrapperTest.java

```diff
@@ -19,8 +19,8 @@
 import org.junit.jupiter.api.Test;
 
 import static io.fabric8.kubernetes.client.utils.internal.OptionalDependencyWrapper.wrapRunWithOptionalDependency;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Instead of hamcrest could you please use assertj.
```suggestion
import static org.assertj.core.api.Assertions.assertThat;
```